### ISSUE TITLE
Refactoring/code review

### ISSUE THREAD

**Issue by kissmarkandras1** [synd1k4t/Iterex#3]
A place to discuss refactoring/standardizing the code

**Comment by synd1k4t** [comment]
Good idea, it should be as simple as putting the source files in folders, and adding the package name after namespace (ex: namespace iterex.entities)
I'll get to it once I figure out how git works in relation to moving files, or you can do it if you know

**Comment by synd1k4t** [comment]
Not sure we should be so strict with the underscores and cases, specifically I find it more convenient to have all variables start with lowercase and classes and methods with uppercase, but it is a good idea to try and follow this

**Comment by vantoan19** [comment]
> Not sure we should be so strict with the underscores and cases, specifically I find it more convenient to have all variables start with lowercase and classes and methods with uppercase, but it is a good idea to try and follow this

Yes, i also feel using camelCase is more convenient but this way makes us easier to know which one is parameter, which one is private attribute (field in C#) and which one is property

**Comment by synd1k4t** [comment]
The one about private attributes is understandable, but for public ones it really feels weird to capitalize variables, though I guess we can try
